README.md

```diff
@@ -93,14 +104,14 @@ You need a runtime library with annotations and code that will run benchmarks on
 
 ```groovy
 repositories {
-    maven { url 'https://dl.bintray.com/kotlin/kotlinx' }
+    mavenCentral()
```

Comment:
AFAIK it doesn't require to be configured explicitly.

Reply:
Why not?
Of course, it has to.
In which repository should gradle look if not declared?

Reply:
ok, you're right

Reply:
Sorry if I sounded a little bit rude with that 🙈

On the other side, this may can removed as well. Depending on how much "copy paste code" you want to provide.
Most open source projects just provide the code snipped of the dependency declaration. Since you mention above already that it is published to mavenCentral, a consumer of the library/plugin could be so smart to know how to declare that repository in Gradle by themself.